### ISSUE TITLE
defaultSeverity set to error does not work

### ISSUE THREAD

**Issue by casper123** [palantir/tslint#4749]
### Bug Report

- __TSLint version__: 5.16.0
- __TypeScript version__: 3.1.6
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
await this.navCtrl.push("DocumentPage", {fromTP: true, jobId: object.jobid});
```

with `tslint.json` configuration:

```
{
    "extends": [
        "tslint:recommended",
        "tslint-config-airbnb",
        "tslint-clean-code"
    ],
    "rulesDirectory": [
        "node_modules/tslint-eslint-rules/dist/rules"
    ],
    "defaultSeverity": "error",
    "rules": {}
}
```

#### Actual behavior
Running lint shows the issue as warning. 
```
[14:59:43]  tslint: src/pages/tech-portal/tech-portal.ts, line: 233 
            A space is required after '{' 

     L232:    await this.showAlert(JTMAlertType.WARNING, "Job Task Cannot Be Closed", isClosable);
     L233:    await this.navCtrl.push("DocumentPage", {fromTP: true, jobId: object.jobid});
     L234:  } else {

```
#### Expected behavior
The lint should show error instead of warning as I've set **defaultSeverity** to **error**

**Comment by adidahiya** [comment]
How do you know it is showing as a warning? I can't tell from your console output. Which formatter are you using?

**Comment by casper123** [comment]
Here is the one using pro terminal theme. The error are shown in red while warnings are yellow
![Screenshot 2019-05-31 at 3 31 55 AM](https://user-images.githubusercontent.com/2751821/58669185-b8a7b180-8354-11e9-9040-b052f352d677.png)

**Comment by casper123** [comment]
By bad, Actually I'm using ionic framework. By default ionic uses` ionic-app-scripts lint`, changing it to use tslint like this `"lint": "tslint --format verbose --project tsconfig.json --config tslint.json",` solved the issue :)

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
